pkgs/by-name/hy/hyprshot/package.nix

```
@@ -0,0 +1,46 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, hyprland
+, jq
+, grim
+, slurp
+, wl-clipboard
+, libnotify
+, makeWrapper
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "hyprshot";
+  version = "1.2.3";
+
+  src = fetchFromGitHub {
+    owner = "Gustash";
+    repo = "hyprshot";
+    rev = finalAttrs.version;
+    hash = "sha256-sew47VR5ZZaLf1kh0d8Xc5GVYbJ1yWhlug+Wvf+k7iY=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 hyprshot -t "$out/bin"
+    wrapProgram "$out/bin/hyprshot" \
+      --prefix PATH ":" ${lib.makeBinPath[
+        hyprland jq grim slurp wl-clipboard libnotify
+      ]}
```

```suggestion
      --prefix PATH ":" ${lib.makeBinPath [
          hyprland jq grim slurp wl-clipboard libnotify
        ]}
```
Makes a little clearer that the list is an argument to `lib.makeBinPath` and not `wrapProgram`